act_ucs.md

```diff
@@ -4,43 +4,72 @@ This file contains the SWARM ACT user stories.
 
 ZenHub Link: [SpDevTeam](https://app.zenhub.com/workspaces/spdevteam-6544d91246b817002d853e69/board)
 
+### Context & definitions
+- Content is created as a 'channel', i.e. as a feed with a specific topic.
+- The content is uploaded to the feed, and new version of the content could be uploaded any time.
+- The content can be accessed by the publisher and any viewers if it does not have ACT defined.
+- The content can be accessed by the publisher and any viewers if it has ACT defined and the viewer is in the grantees list.
+- A publisher is a user who is uploading new content.
+- A viewer is a user who is accessing content.
+- When a viewer is losing access it means the user-specific encryption key was changed.
+- The publisher should be able to modify the ACT, using the same private key, but on multiple nodes in parallel.
```

Comment:
What does 'multiple nodes' mean in this context?

Reply:
So this is something that was mentioned on the workshop as an edge case. When someone is using a node with a private key, changing the ACT, but then another node instance with the same private key is editing the ACT at the same time or some time later.
We can discuss this and we can remove it if it's less likely or we don't want to consider it for now.